examples/mc_npt_spce.rs

```diff
@@ -0,0 +1,125 @@
+// Lumol, an extensible molecular simulation engine
+// Copyright (C) 2015-2016 G. Fraux — BSD license
+extern crate lumol;
+extern crate lumol_input as input;
+
+use lumol::Logger;
+use lumol::sys::{UnitCell, System, Trajectory};
+use lumol::units;
+use lumol::sim::Simulation;
+use lumol::energy::{LennardJones, PairInteraction, PairRestriction, NullPotential, Ewald};
+use lumol::sim::mc::{MonteCarlo, Translate, Rotate, Resize};
+use lumol::out::{EnergyOutput, PropertiesOutput, TrajectoryOutput}; 
+
+use std::path::Path;
+use std::sync::{Once, ONCE_INIT};
+static START: Once = ONCE_INIT;
+
+fn get_system() -> System {
+    let data_dir = Path::new(file!()).parent().unwrap().join("data");
+    let configuration = data_dir.join("spce.xyz");
+    let mut system = Trajectory::open(configuration)
+                                .and_then(|mut traj| traj.read_guess_bonds())
+                                .unwrap();
+    system.set_cell(UnitCell::cubic(20.0));
+
+    // Add intermolecular interactions
+    let lj = Box::new(LennardJones{
+        sigma: units::from(3.16555789, "A").unwrap(),
+        epsilon: units::from(0.65017254, "kJ/mol").unwrap()
+    });
+    let mut pairs = PairInteraction::new(lj, 6.0);
+    // Restrict interactions to act only between different molecules.
+    pairs.set_restriction(PairRestriction::InterMolecular);
+    pairs.enable_tail_corrections();
+    system.interactions_mut().add_pair("O", "O", pairs);
+
+    // Add bonds: we use fixed bond lengths assuming
+    // the equilibrium bond length. This means that both
+    // energy as well as virial for the bond potential are
+    // zero. Hence, we use a `NullPotential`.
+    let bond = Box::new(NullPotential{});
+    system.interactions_mut().add_bond("O", "H", bond);
+    let angle = Box::new(NullPotential{});
+    system.interactions_mut().add_angle("H", "O", "H", angle);
+
+    // Set charges
+    let h = 0.42380;
+    let o = - 2.0 * h;
+    for p in &mut system {
+        match p.name() {
+            "H" => p.charge = h,
+            "O" => p.charge = o,
+            _ => panic!("Particle {}", p.name()),
```

Comment:
Maybe use a better message here ("Unknown particle in charge setting")? And replace `p` by `particle`?